### DISCUSSION TITLE
How to get gender specific avatars from the apis

### DISCUSSION THREAD

**Discussion by ghost** [dicebear/dicebear#181] [Q&A] [answered]
How to get gender specific avatars from the apis
Like we could pass a parameter which would generate a gender Specific avatar

**Comment by FlorianKoerner** [comment]
Hi @volcareso

You have to define for yourself what is female or male for you. This issue should help you:

- https://github.com/dicebear/dicebear/issues/21
- https://github.com/dicebear/dicebear/discussions/179

**Comment by ghost** [comment]
> You have to define for yourself what is female or male for you.

Hair length 

I am using big-smile one

**Comment by ghost** [comment]
> This issue should help you

I saw them but it's too complex

**Comment by ghost** [comment]
Can you please give me config of the url where hair is short only

And another one where hair is long only.

Please

**Comment by FlorianKoerner** [comment, marked as answer]
shortHair
https://avatars.dicebear.com/api/big-smile/seed.svg?hair=shortHair,mohawk,curlyShortHair,shavedHead

longHair
https://avatars.dicebear.com/api/big-smile/seed.svg?hair=bunHair,straightHair,halfShavedHead,bangs,wavyBob,bowlCutHair,curlyBob,froBun,braids

See all options here:
- https://www.figma.com/file/3Te9c70BX2Aj4IUP35sWhF/big-smile?node-id=8%3A371
- https://avatars.dicebear.com/styles/big-smile#style-options
